### ISSUE TITLE
Strange NPM errors with Meteor

### ISSUE THREAD

**Issue by aldofunes** [sstur/react-rte#55]
Hi, I am using Meteor 1.3 with React. I installed react-rte and everything works wonderfully. Great job with this RTE!

However, I am having this annoying error being logged to my server's console:
````
Unable to resolve some modules:

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./Button.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js(web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./ButtonWrap.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./IconButton.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./ButtonGroup.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./InputPopover.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./Dropdown.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./EditorToolbar.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../node_modules/raw-loader/index.js!./Draft.global.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

"!!./../node_modules/css-loader/index.js?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./RichTextEditor.css" in ~/MeteorProject/node_modules/react-rte/dist/react-rte.js (web.browser)

If you notice problems related to these missing modules, consider running:

  meteor npm install --save !!.
````

I tried `npm install css-loader` both locally and globally to no avail.

I repeat, It works, but I would love to make this error go away. Any ideas?

**Comment by NotBrianZach** [comment]
you might try posting an issue or asking around a meteor channel or irc as well. Doesn't look offhand, to me, to be a problem with react-rte per se.

**Comment by sstur** [comment]
Hmm, this seems to be the same issue as #32 which I'm not sure how to fix. It's related to CSS Modules which we use Webpack loaders to handle.

I am not sure why meteor is trying to resolve those modules, since everything should be in the dist/react-rte.js file, already bundled. But evidently browserify has similar issues (maybe Meteor uses browersify or something similar under the hood).

I will reach out to the CSS Modules team and ask if they know anything about this.

**Comment by aldofunes** [comment]
Makes sense. It might have something to do with Meteor not using Webpack as its build engine. I will dig a bit more into it

**Comment by sstur** [comment]
I'm going to close this in favor of #93. It might already be fixed by #80.
